### PR TITLE
translate namespace-v1 page

### PR DESCRIPTION
**Pull request by syxunion**
As title 
Fixex #33931

### CODE REVIEW

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 1]
```diff
@@ -0,0 +1,807 @@
...
+api_metadata:
+  apiVersion: "v1"
+  import: "k8s.io/api/core/v1"
+  kind: "Namespace"
+content_type: "api_reference"
+description: "Namespace 为 NAMES 提供了一个作用域。"
+title: "Namespace"
+weight: 2
+auto_generated: true
+---
```
Comment (Sea-n) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:11]: Please add the original metadata to the comment

Ref: https://kubernetes.io/zh/docs/contribute/localization_zh/#frontmatter-%E7%9A%84%E5%A4%84%E7%90%86

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 2]
```diff
@@ -0,0 +1,807 @@
...
+api_metadata:
+  apiVersion: "v1"
+  import: "k8s.io/api/core/v1"
+  kind: "Namespace"
+content_type: "api_reference"
+description: "Namespace 为 NAMES 提供了一个作用域。"
+title: "Namespace"
+weight: 2
+auto_generated: true
```
Comment (Sea-n) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:10]: The localized pages are not `auto_generated`, please just remove this line.

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 3]
```diff
@@ -0,0 +1,807 @@
...
+
+  - **conditions.status** (string), required
+
+    Status of the condition, one of True, False, Unknown.
+-->
+  **NamespaceCondition 包含命名空间状态的细节。**
+
+  - **conditions.status** (string), 必需
+
+    条件的状态，为True，或False，或Unknown。
```
Comment (Sea-n) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:89]:
```suggestion
    條件的狀態可以是 True、False 或 Unknown。
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 4]
```diff
@@ -0,0 +1,807 @@
...
+<!--
+  *NamespaceCondition contains details about state of namespace.*
+
+  - **conditions.status** (string), required
+
+    Status of the condition, one of True, False, Unknown.
+-->
+  **NamespaceCondition 包含命名空间状态的细节。**
+
+  - **conditions.status** (string), 必需
```
Comment (Sea-n) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:87]:
```suggestion
  - **conditions.status** (string)，必需
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 5]
```diff
@@ -0,0 +1,807 @@
...
+
+    命名空间控制器条件类型。
+    
+  - **conditions.lastTransitionTime** (Time)
+
+    <a name="Time"></a>
+<!--
+    *Time is a wrapper around time.Time which supports correct marshaling to YAML and JSON.  Wrappers are provided for many of the factory methods that the time package offers.*
+-->
+    **Time 是时间的包装。Time 支持对YAML和JSON进行正确封送。为时间包提供的许多函数方法提供了包装器**
```
Comment (Sea-n) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:107]: There should be a space between Mandarin and English word.
Ref: https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%8E%92%E7%89%88%E6%A0%BC%E5%BC%8F

```suggestion
    **Time 是 time.Time 的包装，支持对 YAML 和 JSON 进行正确封送，为 time 包的许多函数方法提供了包装器。**
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 6]
```diff
@@ -0,0 +1,807 @@
...
+<hr>
+
+- **apiVersion**: v1
+
+- **kind**: Namespace
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+<!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+-->
```
Comment (Sea-n) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:31]: Without indention spaces, the following paragraph won't be at the same level with list item.
```suggestion
  <!--
  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
  -->
```

**Review by Sea-n — COMMENTED** [review]
Welcome to contributing Kubernetes website!
I've added some comment for your translation, please take a look.

Here is your preview for this page: [Netlify](https://deploy-preview-33982--kubernetes-io-main-staging.netlify.app/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1/)

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 7]
```diff
@@ -0,0 +1,807 @@
...
+
+`apiVersion: v1`
+
+`import "k8s.io/api/core/v1"`
+
+## Namespace {#Namespace}
+<!--
+Namespace provides a scope for Names. Use of multiple namespaces is optional.
+-->
+Namespace 为 Names 提供了作用域。使用多个命名空间是可选的。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:21]: 这里的 Name 虽然大写，还是翻译为好，因为 Name 不是一个单独定义的类型。

```suggestion
Namespace 为名字提供作用域。使用多个命名空间是可选的。
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 8]
```diff
@@ -0,0 +1,807 @@
...
+
+- **apiVersion**: v1
+
+- **kind**: Namespace
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+<!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+-->
+  标准对象的元数据。 更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:32]:
```suggestion
  标准的对象元数据。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 9]
```diff
@@ -0,0 +1,807 @@
...
+
+- **spec** (<a href="{{< ref "../cluster-resources/namespace-v1#NamespaceSpec" >}}">NamespaceSpec</a>)
+<!--
+  Spec defines the behavior of the Namespace. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+-->
+  Spec 定义了 Namespace 的行为。 更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:38]:
```suggestion
  spec 定义了 Namespace 的行为。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 10]
```diff
@@ -0,0 +1,807 @@
...
+
+- **status** (<a href="{{< ref "../cluster-resources/namespace-v1#NamespaceStatus" >}}">NamespaceStatus</a>)
+<!--
+  Status describes the current status of a Namespace. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+-->
+  Status 描述了当前 Namespace 的状态。 更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:44]:
```suggestion
  status 描述了 Namespace 的当前状态。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 11]
```diff
@@ -0,0 +1,807 @@
...
+-->
+NamespaceSpec 用于描述 Namespace 的属性。
+
+<hr>
+
+- **finalizers** ([]string)
+<!--
+  Finalizers is an opaque list of values that must be empty to permanently remove object from storage. More info: https://kubernetes.io/docs/tasks/administer-cluster/namespaces/
+-->
+  Finalizers 是一个不透明的值列表，必须为空时才能从存储中永久删除对象。 更多信息： https://kubernetes.io/docs/tasks/administer-cluster/namespaces/
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:58]:
```suggestion
  finalizers 是一个不透明的值列表，只有此列表为空时才能从存储中永久删除对象。
  更多信息：https://kubernetes.io/docs/tasks/administer-cluster/namespaces/
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 12]
```diff
@@ -0,0 +1,807 @@
...
+
+## NamespaceStatus {#NamespaceStatus}
+<!--
+NamespaceStatus is information about the current status of a Namespace.
+-->
+NamespaceStatus 表示当前 Namespace 的状态信息。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:64]:
```suggestion
NamespaceStatus 表示 Namespace 的当前状态信息。
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 13]
```diff
@@ -0,0 +1,807 @@
...
+
+- **conditions** ([]NamespaceCondition)
+<!--
+  *Patch strategy: merge on key `type`*
+  
+  Represents the latest available observations of a namespace's current state.
+-->
+  **补丁策略：基于 `type` 健合并**
+  
+  表示当前最新可用的命名空间状态。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:75]: Condition 翻译为状况

```suggestion
  表示命名空间当前状态的最新可用状况。
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md (hunk 3 above)
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:89]:
```suggestion
    状况（condition）的状态，取值为 True、False 或 Unknown 之一。
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 14]
```diff
@@ -0,0 +1,807 @@
...
+<!--
+  - **conditions.type** (string), required
+
+    Type of namespace controller condition.
+    
+  - **conditions.lastTransitionTime** (Time)
+-->
+  - **conditions.type** (string), 必需
+
+    命名空间控制器条件类型。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:99]:
```suggestion
    命名空间控制器状况的类型。
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md (hunk 5 above)
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:107]:
```suggestion
    **Time 是对 time.Time 的封装。Time 支持对 YAML 和 JSON 进行正确封包。为 time 包的许多函数方法提供了封装器。**
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 15]
```diff
@@ -0,0 +1,807 @@
...
+    *Time is a wrapper around time.Time which supports correct marshaling to YAML and JSON.  Wrappers are provided for many of the factory methods that the time package offers.*
+-->
+    **Time 是时间的包装。Time 支持对YAML和JSON进行正确封送。为时间包提供的许多函数方法提供了包装器**
+  - **conditions.message** (string)
+
+  - **conditions.reason** (string)
+
+- **phase** (string)
+
+  Phase 是命名空间的当前生命周期阶段。更多信息： https://kubernetes.io/docs/tasks/administer-cluster/namespaces/
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:114]: 英文原文丢失。

```suggestion
  phase 是命名空间的当前生命周期阶段。更多信息：https://kubernetes.io/docs/tasks/administer-cluster/namespaces/
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 16]
```diff
@@ -0,0 +1,807 @@
...
+
+
+- **kind**: NamespaceList
+
+
+- **metadata** (<a href="{{< ref "../common-definitions/list-meta#ListMeta" >}}">ListMeta</a>)
+<!--
+  Standard list metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+-->
+  标准元数据列表。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:133]:
```suggestion
  标准的列表元数据。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 17]
```diff
@@ -0,0 +1,807 @@
...
+
+- **items** ([]<a href="{{< ref "../cluster-resources/namespace-v1#Namespace" >}}">Namespace</a>), required
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:135]:
```suggestion
- **items** ([]<a href="{{< ref "../cluster-resources/namespace-v1#Namespace" >}}">Namespace</a>)，必需
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 18]
```diff
@@ -0,0 +1,807 @@
...
+<!--
+  Items is the list of Namespace objects in the list. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/
+-->
+Items 是列表中的 Namespace 对象列表。更多信息：https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:139]: 不要改缩进。

```suggestion
  items 是列表中的 Namespace 对象列表。更多信息：https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/。
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 19]
```diff
@@ -0,0 +1,807 @@
...
+
+#### 参数
+<!--
+- **name** (*in path*): string, required
+
+  name of the Namespace
+
+- **pretty** (*in query*): string
+-->
+- **name** (**路径参数**): string, 必需
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:168]:
```suggestion
- **name** (**路径参数**)：string，必需
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 20]
```diff
@@ -0,0 +1,807 @@
...
+<!--
+### `list` list or watch objects of kind Namespace
+
+#### HTTP Request
+
+GET /api/v1/namespaces
+
+#### Parameters
+-->
+### `list` 列出或者观察类型为 Namespace 的对象
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:234]:
```suggestion
### `list` 列出或者监测类别为 Namespace 的对象
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 21]
```diff
@@ -0,0 +1,807 @@
...
+-->
+- **continue** (**查询参数**): string
+
+  <a href="{{< ref "../common-parameters/common-parameters#continue" >}}">continue</a>
+
+<!--
+- **fieldSelector** (*in query*): string
+-->
+- **fieldSelector** (**查询参数**): string
+  <a href="{{< ref "../common-parameters/common-parameters#fieldSelector" >}}">fieldSelector</a>
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:260]: 这种链接前后有无空行很重要！！！

**Comment by tengqm** [comment]
请参照 preview （https://deploy-preview-33982--kubernetes-io-main-staging.netlify.app/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1/）修改空行、缩进格式。

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md (hunk 6 above)
Comment (syxunion) [reply content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md]: Thank you for your comments. The translation has been improved according to the review

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md (hunk 21 above)
Comment (syxunion) [reply content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:287]: Thank you for your comments. The translation has been improved according to the review

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md (hunk 7 above)
Comment (tengqm) [reply content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:32]: ?

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 22]
```diff
@@ -0,0 +1,837 @@
...
+
+- **pretty** (*in query*): string
+-->
+- **name** (**路径参数**)：string，必需
+
+  Namespace 的名称
+
+- **pretty** (**查询参数**)：string
+ <a href="{{< ref "../common-parameters/common-parameters#pretty" >}}">pretty</a>
+<!--
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:200]: 前面加空行

File: content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md [hunk 23]
```diff
@@ -0,0 +1,837 @@
...
+  name of the Namespace
+
+- **pretty** (*in query*): string
+-->
+- **name** (**路径参数**)：string，必需
+
+  Namespace 的名称
+
+- **pretty** (**查询参数**)：string
+ <a href="{{< ref "../common-parameters/common-parameters#pretty" >}}">pretty</a>
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md:199]: 这个前后为什么没加空行？

Comment (syxunion) [reply content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md]: 不好意思，没有注意到。已经修改。

**Pull request merged**